dll/win32/version/version.c

```diff
@@ -979,6 +979,14 @@ static BOOL VersionInfo32_QueryValue( const VS_VERSION_INFO_STRUCT32 *info, LPCW
 
     /* Return value */
     *lplpBuffer = VersionInfo32_Value( info );
+
+#ifdef __REACTOS__
+    /* If the wValueLength is zero, then set a UNICODE_NULL only return string.
+     * Use the NULL terminator from the key string for that. This is what Windows does, too. */
```

Comment:
So yes, a test for this would be needed I suppose.
Also, what would happen if we did instead `*lplpBuffer = L"";` ?

I even wonder, whether we could place the fix you're doing here inside the `VersionInfo32_Value` function itself :thinking: